packages/desktop/components/popups/NodeInfoPopup.svelte

```diff
@@ -74,8 +74,20 @@
     ): { localeKey: string; nodeInfoValue: unknown } {
         const nodeInfoTabObject = NODE_INFO_TAB_MAP[_nodeInfoTab]
         const localeKey = nodeInfoTabObject[key].localeKey
-        const nodeInfoValue =
-            key !== 'url' ? resolveObjectPath(nodeInfo, nodeInfoTabObject[key]?.nodeInfoPath, null) : node?.url
+        let nodeInfoValue = ''
+        if (key === 'url') {
+            nodeInfoValue = node.url
+        } else {
+            nodeInfoValue = resolveObjectPath(nodeInfo, nodeInfoTabObject[key]?.nodeInfoPath, null)
+            if (localeKey === 'metrics.referencedRate') {
```

Comment:
I would use the key here and not the `localeKey`

```suggestion
            if (key === 'referencedRate') {
```

Reply:
I would also round to 1 or 2 decimals the 2 first elements
![image](https://github.com/iotaledger/firefly/assets/3624944/b34dd5aa-9255-4ab9-858b-7eca664530cb)